build_projects/shared-build-targets-utils/Utils/StubPackageBuilder.cs

```diff
@@ -0,0 +1,101 @@
+using System;
+using System.Collections.Generic;
+using System.IO;
+using System.Linq;
+using System.Net.Http;
+using System.Text;
+using Microsoft.DotNet.Cli.Build.Framework;
+using Microsoft.WindowsAzure.Storage;
+using Microsoft.WindowsAzure.Storage.Blob;
+
+using static Microsoft.DotNet.Cli.Build.Framework.BuildHelpers;
+
+namespace Microsoft.DotNet.Cli.Build
+{
+    public class StubPackageBuilder
+    {
+        private DotNetCli _dotnet;
+        private string _intermediateDirectory;
+        private string _outputDirectory;
+
+        private bool _stubBitsBuilt = false;
+
+        public StubPackageBuilder(DotNetCli dotnet, string intermediateDirectory, string outputDirectory)
+        {
+            _dotnet = dotnet;
+            _intermediateDirectory = intermediateDirectory;
+            _outputDirectory = outputDirectory;
+        }
+
+        public void GeneratePackage(string packageId, string version)
+        {
+            if (! _stubBitsBuilt)
+            {
+                BuildStubBits(_dotnet, _intermediateDirectory);
+            }
+
+            CreateStubPackage(_dotnet, packageId, version, _intermediateDirectory, _outputDirectory);
+        }
+
+        private void BuildStubBits(DotNetCli dotnet, string intermediateDirectory)
+        {
+            var projectJson = new StringBuilder();
+            projectJson.Append("{");
+            projectJson.Append("  \"dependencies\": { \"NETStandard.Library\": \"1.5.0-rc2-24008\" },");
```

Comment:
Good thing this will go away when we move to the new repo.  Here's another version string that should be updated by our `update-dependencies` script.